pom.xml

```diff
@@ -62,6 +62,7 @@
         <hazelcast.version>4.2.8</hazelcast.version>
 
         <jackson.version>2.15.2</jackson.version>
+        <jackson.path.version>2.4.0</jackson.path.version>
         <jaxb.api.version>4.0.0</jaxb.api.version>
```

Comment:
json.path.version? Jackson and json path are different libraries

Reply:
oh yes, you are right.